march_hardware/src/EtherCAT/EthercatSDO.cpp

```diff
@@ -18,19 +18,36 @@ namespace march4cpp
 int sdo_bit8(int slave, uint32_t index, uint8_t sub, uint8_t value)
 {
   ROS_DEBUG("sdo_bit8: slaveIndex %i, reg 0x%X, subindex %i, value 0x%X", slave, index, sub, value);
-  return ec_SDOwrite(slave, index, sub, FALSE, 1, &value, EC_TIMEOUTRXM);
+  int ReceivedWorkingCounter = ec_SDOwrite(slave, index, sub, FALSE, 1, &value, EC_TIMEOUTRXM);
```

Comment:
Name your local variables **under_scored** (https://wiki.ros.org/CppStyleGuide#Variables)
```suggestion
  int received_working_counter = ec_SDOwrite(slave, index, sub, FALSE, 1, &value, EC_TIMEOUTRXM);
```